src/EditorFeatures/Core/Implementation/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -98,7 +99,18 @@ internal static class GoToDefinitionHelpers
         public static bool TryGoToDefinition(
             ImmutableArray<INavigableItem> navigableItems,
             string title,
-            Workspace workspace,
+            Workspace originalWorkspace,
```

Comment:
if this is just to change option temporary, how about something like TryGoToDefintionWithProvisionalTabIfPossible and give option in?